function_calling.ipynb

```diff
@@ -0,0 +1,215 @@
+{
```

Comment:
<p>It'd be great if you can add more description that explains what Function Calling is. Something like this :</p><blockquote>you can describe functions and have the model intelligently choose to output a JSON object containing arguments to call one or many functions. The model does not call the function, but it generates JSON that you can use to call the function in your code.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UpstageAI/cookbook/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>